db/db_iter.cc

```diff
@@ -264,6 +264,7 @@ class DBIter final: public Iterator {
 
   Status status_;
   IterKey saved_key_;
+  ParsedInternalKey tmp_ikey_;
```

Comment:
I suggest to rename this to something more meaningful, as this is a class field and not a function-local variable.

Reply:
Do you have any suggestion? Or I can add comments explaining it.

Reply:
May be call it just `ikey_`, and add a comment about the overhead.